include/glaze/json/schema.hpp

```diff
@@ -537,10 +612,15 @@ namespace glz
                   }
                   else if constexpr (std::is_convertible_v<additional_data_type, schema>) {
                      ref_val = get<comment_index>(item);
+                     validate_ref<name_v<val_t>>();
                      ref_val.ref = join_v<chars<"#/$defs/">, name_v<val_t>>;
                   }
                }
 
+               if (!def.type) {
```

Comment:
I don't remember why this was needed but it was required for our frontend at the time which I do not currently have access to. This is trivial change though so I don't see a problem with it. Please comment if you feel otherwise

Reply:
I don't see a problem with it